$NetBSD: patch-src_common_meson.build,v 1.1 2019/07/17 10:53:22 nia Exp $

Make generated headers an explicit dependency of targets using
hexchat_common to force the headers to be generated in time.

--- src/common/meson.build.orig	2018-08-29 21:13:25.000000000 +0000
+++ src/common/meson.build
@@ -109,6 +109,7 @@ hexchat_common = static_library('hexchat
 )
 
 hexchat_common_dep = declare_dependency(
+  sources: [textevents] + marshal,
   link_with: hexchat_common,
   include_directories: common_includes,
   compile_args: common_cflags,
